app/Http/Transformers/PostTransformer.php

```diff
@@ -30,12 +30,15 @@ public function transform(Post $post)
             'signup_id' => $post->signup_id,
             'northstar_id' => $post->northstar_id,
             'media' => [
-
                 'url' => $post->getMediaUrl(),
                 'original_image_url' => $post->url,
                 'caption' => $post->caption,
             ],
             'tags' => $post->tagSlugs(),
+            'reactions' => [
+                'reacted' => $post->reactions->isNotEmpty() ? true : false,
+                'total' => $post->reactions_count,
```

Comment:
Related question: do we always load a count of reactions? If not, might be good to conditionally include this field only if `reactions_count` is set on the post.

Reply:
oooo this is a great question and I don't think that we do which would definitely break this